packages/plugin-http-client/src/rest/AbstractEntity.ts

```diff
@@ -0,0 +1,194 @@
+// @ts-ignore
+import { clone } from '@ima/helpers';
+
+import { Mapper } from './mapper/Mapper';
+
+export type MapperItem = {
+  mapper: Mapper;
+  newKey: string;
+};
+
+/**
+ * The base class for typed REST API entities. Usage of typed entities may be
+ * optional and is dependent on the specific implementation of the REST API
+ * client.
+ */
+export class AbstractEntity {
```

Comment:
Maybe rename to `BaseEntity` it can be used directly, without being extended (abstract)

Reply:
Added interface Entity and renamed to BaseEntity.